src/ui/shared/WindowContext.ts

```diff
@@ -0,0 +1,50 @@
+export type WindowType = 'popup' | 'tab' | 'dialog';
+export type WindowLayout = 'column' | 'page';
+
+export enum WindowParam {
+  windowType = 'windowType',
+  windowLayout = 'windowLayout',
+  mode = 'mode',
```

Comment:
Should we move work with mode to this file too?

Reply:
We don't really access the `mode` parameter anywhere except for `useBackupContext`, where it needs to be checked explicitly (using equality comparison) to help with type inference since the `BackupContext` is kind of a discriminated union

Reply:
✅ Added `appMode` (prev. `context`) to the `WindowContext` class + extra functions:
```
  isOnboardingMode() {
    return this.params.appMode === 'onboarding';
  }

  isWalletMode() {
    return this.params.appMode === 'wallet';
  }
```